fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -386,6 +386,8 @@
 		span_notice("The meat sprouts digits and becomes [beefboy]'s new [new_bodypart.name]!"),
 		span_notice("The meat sprouts digits and becomes your new [new_bodypart.name]!"))
 	new_bodypart.attach_limb(beefboy)
+	new_bodypart.update_limb(is_creating = TRUE)
+	beefboy.update_body_parts()
```

Comment:
Looking into it, ``attach_limb()`` calls ``new_limb_owner.update_body()`` which calls ``update_body_parts(is_creating)``.

is it update_limbs that's the problem here?

Reply:
i think so, i tried adding update_limb by itself but it wouldnt work